index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="./index.js"></script>
+    <title>API lab</title>
+</head>
+<body>
+    <h1 id="jokes">Joke 1</h1>
+    <button id= "punchline">Joke Card</button>
+    <button type="submit"  onClick="refreshPage()">Refresh Button</button>
+    <div>
+        <h3 id="jokes3">Joke by type</h3>
+        <button id= "type1" onClick="generalJoke()">General</button>
```

Comment:
Adding an event listener to the button would be a better way to have the button setup